notebooks/time-to-merge-prediction/workshop/00_read_env.ipynb

```diff
@@ -0,0 +1,307 @@
+{
```

Comment:
<p>Please lets not print any credentials. This is not good practice. People can print creds and its easy to make it public this way.</p><p>@suppathak instead can you add a code block to test out boto3 connection, or print bucket objects in a way to easily verify that the creds are right?</p><p> </p><p>Maybe you can place a file on the bucket and once people are able to find/print that file we can verify that the creds are right</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/564/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>